cirq-ionq/cirq_ionq/ionq_client.py

```diff
@@ -263,6 +265,34 @@ def list_calibrations(
             params['end'] = int((end - epoch).total_seconds() * 1000)
         return self._list('calibrations', params, 'calibrations', limit, batch_size)
 
+    def api_headers(self, api_key: str):
+        """API Headers needed to make calls to the REST API.
+
+        Args:
+            api_key: The key used for authenticating against the IonQ API.
+
+        Returns:
+            dict[str, str]: A dict of :class:`requests.Request` headers.
+        """
+        return {
+            'Authorization': f'apiKey {api_key}',
+            'Content-Type': 'application/json',
+            'User-Agent': self._user_agent(),
+        }
+
+    def _user_agent(self):
+        """Generates the user agent string which is helpful in identifying
+        different tools in the internet. Valid user-agent ionq_client header that
+        indicates the request is from cirq_ionq along with the system, os,
+        python,libraries details.
+
+        Returns:
+            str: A string of generated user agent.
+        """
+        cirq_version_string = f'cirq/{cirq_version}'
+        python_version_string = f'python/{platform.python_version()}'
+        return f'User-Agent: {cirq_version_string} ({python_version_string})'
```

Comment:
Checked that `platform.python_version` doesn't leak OS info (it returns 'major.minor.patchlevel'), so this is much better. Minor suggestion to improve readability of the user-agent of any human actually relying on it:

```python
return f'User-Agent: cirq {cirq_version_string} (python {python_version_string})'
```

This changes current `User-Agent: 1.2.0.dev (3.10.9)` to `User-Agent: cirq 1.2.0.dev (python 3.10.9)` which is easier to interpret without looking at the code.

Reply:
agreed that looks much better ill change that now

Reply:
Oh, you're right of course. I misread the strings.